lib/logitech_receiver/receiver.py

```diff
@@ -99,13 +105,21 @@ def __init__(self, receiver, number, link_notification=None):
 				# unifying protocol not supported, must be a Nano receiver
 				device_info = self.receiver.read_register(_R.receiver_info, 0x04)
 				if device_info is None:
-					_log.error("failed to read Nano wpid for device %d of %s", number, receiver)
-					raise _base.NoSuchDevice(number=number, receiver=receiver, error="read Nano wpid")
+					#ex100 receiver, fill fake device_info with known wpid's
+					if ((receiver.product_id == 'c517') and ((number==1) or (number==3))):
```

Comment:
Are we sure that check in line 109 would not match any other case? I believe C517 is used by other devices other than EX100.

Reply:
I can't find any other method of detect EX100 devices. Yes, i find notices that this usb id also related to 'LX710 Cordless Desktop Laser' and 'Deluxe 660' keyboard and mouse combos and partially discussed in #109. But I don't have this devices and can't find better method. Anyway this old devices have same receiver and we can hope what they will be compatible with EX100.

Reply:
Agreed